tests/provision/container/alpine/test.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+USER="tester"
+
+rlJournalStart
+    rlPhaseStartSetup
+        # Directories
+        rlRun "tmp=\$(mktemp -d)" 0 "Create tmp directory"
+        rlRun "run=\$(mktemp -d)" 0 "Create run directory"
+        rlRun "pushd $tmp"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Test vanilla alpine without bash"
+        rlRun -s "tmt run --all --id $run --verbose --scratch \
+            provision --how container --image docker.io/alpine \
+            execute --how tmt --script whoami" 2
+        rlAssertGrep "fail: Bash is required on the guest." $rlRun_LOG
+    rlPhaseEnd
+
+    rlPhaseStartTest "Test alpine with bash"
+        rlRun -s "tmt run -vv --all --id $run --verbose --scratch \
+            provision --how container --image localhost/alpine:tmt-tests \
+            execute --how tmt --script whoami" 0
+        rlAssertGrep "out: root" $rlRun_LOG
+    rlPhaseEnd
```

Comment:
I'd recommend adding a test (or a phase) for the `prepare/install` plugin, with at least one `require` package - that was the issue, right?

Reply:
yeah, true true

Reply:
I plan to add this test case to `tests/prepare/install`, among other tests checking various distros, so I suppose you can ignore this particular suggestion.